pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/NamespacesBase.java

```diff
@@ -2751,19 +2751,19 @@ protected void internalSetMaxSubscriptionsPerTopic(Integer maxSubscriptionsPerTo
         internalSetPolicies("max_subscriptions_per_topic", maxSubscriptionsPerTopic);
     }
 
-    protected void internalSetMaxUnackedMessagesPerSubscription(int maxUnackedMessagesPerSubscription) {
+    protected void internalSetMaxUnackedMessagesPerSubscription(Integer maxUnackedMessagesPerSubscription) {
         validateNamespacePolicyOperation(namespaceName, PolicyName.MAX_UNACKED, PolicyOperation.WRITE);
         validatePoliciesReadOnlyAccess();
-
+        if (maxUnackedMessagesPerSubscription != null && maxUnackedMessagesPerSubscription < 0) {
```

Comment:
Currently, we use `null` to represent the absence of policy. How the users able to disable this policy for a specific namespace or topic?

For example, if the broker enabled the `maxUnackedMessagesPerSubscription` but we want to disable it in the namespace level or topic level.

Reply:
Just set it to 0, which is consistent with the description in broker.conf, but here it just cannot be set less than 0